runtime/queries/kotlin/injections.scm

```diff
@@ -0,0 +1,32 @@
+(comment) @comment
```

Comment:
nvim does injections very differently, helix is very similar to [tree-sitter's way](https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection), so this one would be

```suggestion
((comment) @injection.content
 (#set! injection.language "comment"))
```